src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -15,5 +15,7 @@
     public static final Prefix PREFIX_PREV_DATE_MET = new Prefix("d/");
     public static final Prefix PREFIX_SALARY = new Prefix("s/");
     public static final Prefix PREFIX_INFO = new Prefix("i/");
+    public static final Prefix PREFIX_MEETING_DATE = new Prefix("d/");
```

Comment:
Just to check, you are using the same prefix as the `prevDateMeet`?

Reply:
Yes, because we're using different commands, there is no conflict. prevDateMeet only works with add and edit. Scheduling meeting uses meet command